.evg.yml

```diff
@@ -252,6 +285,36 @@ functions:
         permissions: public-read
         display_name: mongodb-jdbc-${MDBJDBC_VER}.sbom.json
 
+  "generate and upload third-party-license":
+    - command: shell.exec
+      type: test
+      params:
+        shell: bash
+        working_dir: mongo-jdbc-driver
+        script: |
+          ${PREPARE_SHELL}
+          set -o errexit
+
+          echo "------------------------------------"
+          echo "Generating THIRD-PARTY-LICENSE.txt"
+          echo "------------------------------------"
+          ./gradlew clean generateLicenseReport || true
```

Comment:
Let's move this as part of `build jdbc driver` and modify the renderer to output the file in the resources directory so that it's included in the jar file. Where in the jar, I think doesn't matter. As long as it's inside, we are fine.

You can then also get rid of the separate Evergreen tasks because I don't think that we need to publish it to S3 since the jar inside will be published already.